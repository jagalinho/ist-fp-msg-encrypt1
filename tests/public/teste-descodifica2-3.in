chave = (('A', 'B', 'C', 'D'), ('E', 'F', 'G', 'H'), ('I', 'J', 'L', 'M'), ('N', 'O', 'P', 'Q'), ('R',))
cad_encriptada = '11XX300300231030XX31XXXX0300XX3240311240002300020031'
cad_desencriptada = descodifica2(cad_encriptada, chave)
cad_desencriptada
