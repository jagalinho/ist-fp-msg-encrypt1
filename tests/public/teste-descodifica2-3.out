'F?NDAMEN?O??DA?PROGRAMACAO'
